auto_scan.py

```diff
@@ -144,8 +149,8 @@ def scan_containers(lw_client, containers, scanned_container_cache):
             print(f'Scanning {container_registry}/{container_repository} with tag "{container["TAG"]}" ({i})')
 
             executor_tasks.append(executor.submit(
-                initiate_container_scan, lw_client, container_registry, container_repository, container['TAG']
-            ))
+                    initiate_container_scan, lw_client, container_registry, container_repository, container['TAG']
+                ))
```

Comment:
Removing added whitespace.

```suggestion
                initiate_container_scan, lw_client, container_registry, container_repository, container['TAG']
            ))
```